src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -127,6 +131,15 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
 
       <p className="PosterPreview__title">{title}</p>
 
+      <img
+        src={thumbnailUrl}
+        alt={title}
+        onError={(e) => {
+          e.currentTarget.src = DEFAULT_POSTER_THUMBNAIL_IMG;
+        }}
+        style={{ borderRadius: "15px" }}
+      />
```

Comment:
Having this function inline like this will cause re-renders. It should be `useCallback`'d.

Also, we shouldn't be using inline styles like this (and it will also cause re-renders). It should be made as a new class in the SCSS following BEM naming patterns.

Reply:
Why are we using an `img` here? If we used a `div` or similar then we wouldn't need a hacky workaround with the default image, which is basically just a solid block of colour:

![image](https://user-images.githubusercontent.com/753891/122496137-9ada4180-d02e-11eb-9a54-c005ae34c2c6.png)

We can use the CSS `background-image` property on a `div`, and also set the `background-color`, which is used as a fallback when there is no image/it fails to render/etc.

I believe there should be examples of this already in the codebase. They'll use `useCss` and CSS `--properties` to pass the URL of the image from the JS into SCSS. This will then be combined with the `classNames` helper to apply the variables + specific class for the element to the `className` in the JSX. There are standard naming patterns for these, so we should follow suit. The properties tend to be set on the main container of component.

Reply:
`useCss` + `classNames` pattern: https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx#L26-L33

Using a `div`: https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx#L41

With SCSS styling: https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.scss#L32

Just seems to need a `background-colour` added as well.